crates/router/src/routes/payouts.rs

```diff
@@ -1,42 +1,182 @@
 use actix_web::{
     body::{BoxBody, MessageBody},
-    HttpResponse, Responder,
+    web, HttpRequest, HttpResponse, Responder,
 };
 use router_env::{instrument, tracing, Flow};
 
+use super::app::AppState;
+use crate::{
+    core::payouts::*,
+    services::{api, authentication as auth},
+    types::api::payouts as payout_types,
+};
+
+/// Payouts - Create
+#[utoipa::path(
+    post,
+    path = "/payouts/create",
+    request_body=PayoutCreateRequest,
+    responses(
+        (status = 200, description = "Payout created", body = PayoutCreateResponse),
+        (status = 400, description = "Missing Mandatory fields")
+    ),
+    tag = "Payouts",
+    operation_id = "Create a Payout",
+    security(("api_key" = []))
+)]
 #[instrument(skip_all, fields(flow = ?Flow::PayoutsCreate))]
-// #[post("/create")]
-pub async fn payouts_create() -> impl Responder {
-    let _flow = Flow::PayoutsCreate;
-    http_response("create")
+pub async fn payouts_create(
+    state: web::Data<AppState>,
+    req: HttpRequest,
+    json_payload: web::Json<payout_types::PayoutCreateRequest>,
+) -> HttpResponse {
+    let flow = Flow::PayoutsCreate;
+    api::server_wrap(
+        flow,
+        state.get_ref(),
+        &req,
+        json_payload.into_inner(),
+        payouts_create_core,
+        &auth::ApiKeyAuth,
+    )
+    .await
 }
 
+/// Payouts - Retrieve
+#[utoipa::path(
+    get,
+    path = "/payouts/retrieve/{payout_id}",
```

Comment:
change the path

Reply:
Why does it need changing?

Reply:
as per the recent change in app.rs the path should be `"/payouts/{payout_id}"` right?